synapse/api/constants.py

```diff
@@ -138,3 +138,6 @@ class LimitBlockingTypes(object):
 
     MONTHLY_ACTIVE_USER = "monthly_active_user"
     HS_DISABLED = "hs_disabled"
+
+
+LabelsField = "org.matrix.labels"
```

Comment:
```suggestion
LABELS_FIELD = "org.matrix.labels"
```

also, comment please. where is this used? ref to a MSC here wouldn't hurt either.

It should probably either be in a `class` named appropriately to its usage, or to give it a longer name.